tests/src/HTMLElement/elements/HTMLDocumentElement.js

```diff
@@ -0,0 +1,29 @@
+/*global test*/
+import assert from 'proclaim';
+import {readFileSync} from 'fs';
+
+const expect = assert.strictEqual;
+const throws = assert.throws;
+
+const lib = '../../../../lib/';
+
+const Document = require(lib + 'Document');
+const Doctype = require(lib + 'Doctype');
+
+const fileText = readFileSync(__dirname + '/../../../data/linkstest.html', { encoding: 'utf-8' });
```

Comment:
I don't think you need this big html to create the test below, right ? What you want to test, if I got it right, is:

- `document.documentElement.innerHTML = '<html></html>';`
- `document.documentElement.innerHTML = '<html><body>test</body></html>';`
- `document.documentElement.innerHTML = '<html><head><title>test</title></head></html>';`
- `document.documentElement.innerHTML = '<html><head><title>test</title></head><body>test</body></html>';`

And copy.

What do you think ?